tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -197,6 +197,26 @@ bool TypeTranslator::isScalarType(QualType type, QualType *scalarType) {
   return isScalar;
 }
 
+bool TypeTranslator::isRWByteAddressBuffer(QualType type) {
```

Comment:
I'll keep these two methods separate since the Store, Store2, Store3, Store4 will need to check that they work only for RWByteAddressBuffer.